docusaurus.config.js

```diff
@@ -143,6 +143,14 @@ const config = {
             to: "/blog/the-opentofu-fork-is-now-available",
             from: "/fork",
           },
+          {
+            from: "/cli/install/apt",
```

Comment:
These URLs need to be prefixed with `/docs`.

Reply:
Good catch, thanks!